### PR TITLE
Issue/130 line up factory

### PR DESCRIPTION
**Pull request by jordanmaguire**
see the specs!

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,3 +1,4 @@
 source 'http://rubygems.org'
 
 gemspec
+gem 'byebug'
```
Comment (jordanmaguire) [inline Gemfile]: This didn't work via Gemspec dependency. Not sure why

**Comment by BenR1312** [comment]
I dont see anything wrong, but this was a really damn weird one to look at, and i got maybe like 20% of it

**Comment by psyren89** [comment]
Some examples of how this works would be nice.

**Comment by jordanmaguire** [comment]
> Some examples of how this works would be nice.

Check the specs. It's all there verbatim

**Pull request merged**